integration-tests/features/listing-information/getByIds/id-not-found-multiple-recurrent-brain-neoplasm.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Updated! Renamed the test data files to include both what's being tested and what's in the data. Also renamed to `errors.feature` and associated test data files.

Reply:
The file names still reflect the contents (e.g. `errors-C0000-C0001.json`) instead of the functionality being tested (e.g. `errors-multiple-non-existing-codes.json`).

Reply:
Updated the file names to reflect the functionality tested.